lib/hawkular/hawkular_client.rb

```diff
@@ -2,11 +2,12 @@
 require 'hawkular/alerts/alerts_api'
 require 'hawkular/tokens/tokens_api'
 require 'hawkular/operations/operations_api'
+require 'hawkular/prometheus/prometheus_api'
```

Comment:
I don't like this to be honest. To me it seems like the metrics endpoint should be pluggable, so exposing the metrics provider underlying is binding us on each codebase to that provider, making changes harder. To me, we should just provide a `metrics` endpoint, as a proxy object for each provider (be it hawkular-metrics, be it prometheus).

Reply:
At the moment we need both in parallel as the same gem can be used with hMetrics and P8S

Reply:
Exactly the reason why I think it should be the same endpoint with different providers.

Reply:
No, as they have different models behind.
There is not a pure abstraction model plugable where provider can be hidden.